core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/order/dao/OrderDaoImpl.java

```diff
@@ -78,7 +90,9 @@ public Order readOrderById(final Long orderId) {
         try {
             order = query.getSingleResult();
         } catch (NoResultException e) {
-            LOG.warn(String.format("Could not find order by ID %s", orderId));
+            if (LOG.isWarnEnabled()) {
```

Comment:
@KatrukOV This check is not needed.  The LOG.warn does that internally.  Doing a check with isWarnEnabled is good if you have a block of code that gets executed ahead of the logging taking place.